test/lit.cfg

```diff
@@ -1696,6 +1694,13 @@ elif run_os in ['windows-msvc']:
     config.target_clang =                                                        \
             ('clang++ -target %s %s %s -fobjc-runtime=ios-5.0' %                 \
              (config.variant_triple, clang_mcp_opt, config.target_cc_options))
+
+    # Make sure we pick link.exe from MSVC host toolchain and not from coreutils
+    PATH = config.environment['PATH']
+    PATH_MSVC = ';'.join([p for p in PATH.split(';') if 'MSVC' in p])
```

Comment:
Is a filter like that acceptable? The path should be something like `C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.42.34433\bin\Hostx64\x64\link.exe`